src/app/component/ProductReviewList/ProductReviewList.style.scss

```diff
@@ -0,0 +1,80 @@
+/**
+ * ScandiPWA - Progressive Web App for Magento
+ *
+ * Copyright © Scandiweb, Inc. All rights reserved.
+ * See LICENSE for license details.
+ *
+ * @license OSL-3.0 (Open Software License ("OSL") v. 3.0)
+ * @package scandipwa/base-theme
+ * @link https://github.com/scandipwa/base-theme
+ */
+
+.ProductReviewList {
+    &-Item {
+        border-bottom: 1px solid var(--color-neutral-light);
+        display: flex;
+        flex-wrap: wrap;
+        margin: 0;
+        padding: 3rem 0;
+
+        @include mobile {
+            padding: 2rem 0;
+        }
+
+        &::before {
+            content: none;
+        }
+
+        h4 {
+            width: 100%;
+            margin-bottom: 1rem;
+        }
+    }
+
+    &-RatingSummary {
+        @include mobile {
+            margin-bottom: 1rem;
+        }
+    }
+
+    &-RatingSummaryItem {
+        align-items: center;
+        display: flex;
+        justify-content: space-between;
+
+        .ProductReviewRating {
+            margin: 0 0 .2rem 1rem;
+        }
+    }
+
+    &-ReviewContent {
+        flex: 1 0 0;
+        margin-left: 4rem;
+
+        @include mobile {
+            flex: 0 0 auto;
+            margin-left: 0;
+            width: 100%;
+        }
+
+        p {
+            &:last-of-type {
+                margin-bottom: 2rem;
+
+                @include mobile {
+                    margin-bottom: 1rem;
+                }
+            }
+
+            .TextPlaceholder {
+                &::after {
+
+                }
+            }
+        }
+
+        span {
```

Comment:
Please, do not do that, use BEM. span always inherits, can this font-size be applied to parent?